pages/_profiles/Person/0.3-DRAFT.html

```diff
@@ -0,0 +1,310 @@
+  description: 'Global connetions related to teaching and scholarly work in the past
+    3 years '
+  example: ''
+  expected_types:
+  - Text
+  marginality: Optional
+  property: internatonalActivities
+  type: bioschemas
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: 'Which journals this person is a reeree of '
+  example: ''
+  expected_types:
+  - CreativeWork
+  marginality: Optional
+  property: journalReferee
+  type: bioschemas
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: Keywords to describe the person. These should be single words or short
+    phrases that relate to a person. Keywords or tags used to describe this content.
+    Multiple entries in a keywords list are typically delimited by commas.
+  example: ''
+  expected_types:
+  - Text
+  marginality: Optional
+  property: keywords
+  type: bioschemas
+  type_url: ''
+- bsc_description: ''
+  cardinality: ONE
+  controlled_vocab: ''
+  description: A list of languages spoken by the Person Of a Person, and less typically
+    of an Organization, to indicate a known language. We do not distinguish skill
+    levels or reading/writing/speaking/signing here. Use language codes from the IETF
+    BCP 47 standard.
+  example: ''
+  expected_types:
+  - Language
+  marginality: Optional
+  property: knowsLanguage
+  type: ''
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: 'All or notable publications, presentations, training materials, research
+    outputs or any other body of work attributed to this person. '
+  example: ''
+  expected_types:
+  - CreativeWork
+  marginality: Optional
+  property: works
+  type: bioschemas
+  type_url: ''
+name: Person
+parent_type: Person
+previous_release: ''
+previous_version: 0.2-DRAFT-2019_07_19
+redirect_from: '- "/devSpecs/Person/specification/"
+
+  - "/devSpecs/Person/specification"
+
+  - "/devSpecs/Person/"
+
+  - "/devSpecs/Person"
+
+  - "/specifications/drafts/Person/"
+
+  - "/specifications/drafts/Person"
+
+  - "/specifications/Person/"
+
+  - "/specifications/Person"
+
+  - "/profiles/Person/"
+
+  - "/profiles/Person"'
+spec_info:
+  description: 'A specification for describing people in life sciences The Life Science
+    Person specification provides a way to describe bioscience events on the World
+    Wide Web. It defines a set of metadata and vocabularies, built on top of existing
+    technologies and standards, that can be used to represent the profile information
+    of people in Web pages and applications. The goal of the specification is to make
+    it easier to discover, life science profile information. Version: 0.3-DRAFT. <h3>Summary
+    of Changes</h3><br>Changes since 0.2-DRAFT-2019_07_19:<ul><li>added sameAs property
+    as recommended</li></ul>. '
+  full_example: https://github.com/BioSchemas/specifications/tree/master/Person/examples
```

Comment:
This URL https://github.com/BioSchemas/specifications/tree/master/Person/examples does not exist. 
The one that exists is https://github.com/BioSchemas/specifications/tree/master/Person/jsonld

According to the new tutorial, examples will be under the ```jsonld``` so the GitHub actions should use that path